File: 2-Patron/php/Montura.php
<?php
class Montura
{
  private $nombre;
  private $velocidad;
  private $nivel;

  function __construct($nombre, $velocidad, $nivel)
  {
    $this->nombre = $nombre;
    $this->velocidad = $velocidad;
    $this->nivel = $nivel;
  }

  function getNombre()
  {
    return $this->nombre;
  }

  function setNombre($nNombre)
  {
    $this->nombre = $nNombre;
  }

  function getVelocidad()
  {
    return $this->velocidad;
  }

  function setVelocidad($nVelocidad)
  {
    $this->velocidad = $nVelocidad;
  }

  function getNivel()
  {
    return $this->nivel;
  }

  function setNivel($nNivel)
  {
    $this->nivel = $nNivel
  }

  function incNivel()
  {
    $this->nivel += 1;
  }
}
?>

File: 2-Patron/Java/Montura.java
package abstractmasterwhole;
public class Montura {
    private String nombre;
    private float velocidad;
    private int nivel;
    
    public Montura(String nombre, float velocidad, int nivel){
        this.nombre = nombre;
        this.velocidad = velocidad;
        this.nivel = nivel;
    }
    public String getNombre(){
        return this.nombre;
    }
    public void setNombre(String nombre){
        this.nombre = nombre;   
    }
    public float getVelocidad(){
        return this.velocidad;
    }
    public void setVelocidad(float velocidad){
        this.velocidad = velocidad;
    }
    public int getNivel(){
        return this.nivel;
    }
    public void setNivel(int nivel){
        this.nivel = nivel;
    }
    public int incNivel(){
        this.nivel += 1;
        return this.nivel;
    }
}
